pkg/controller/bastion/actuator_reconcile.go

```diff
@@ -356,6 +357,26 @@ func ensureSecurityGroupRules(log logr.Logger, client openstackclient.Networking
 		log.Info("Unwanted security group rule deleted", "rule", rule.Description, "ruleID", rule.ID)
 	}
 
+	// apply bastion ingress rule in worker node security group
+	for _, ingressPermission := range ingressPermissions {
+		wantedRules = append(wantedRules,
+			//bastion ingress rule in worker node security group
+			IngressAllowSSH(opt, ingressPermission.EtherType, infraStatus.SecurityGroups[0].ID, "", secGroupID),
+		)
+	}
```

Comment:
it seems must be not empty otherwise ` Missing input for argument [EtherType]",`  
```
type CreateOpts struct {
	// Must be "IPv4" or "IPv6", and addresses represented in CIDR must match the
	// ingress or egress rules.
	EtherType RuleEtherType `json:"ethertype" required:"true"`
```
0ea3f840